docs/api/decorators/searchIndexes.md

```diff
@@ -0,0 +1,75 @@
+---
+id: searchIndexes
+title: '@searchIndex'
+---
+
+**Typings:**
+
+```ts
+function searchIndex<T extends BeAnObject = BeAnObject>(description: SearchIndexDescription): ClassDecorator
+```
+
+**Parameters:**
+
+| Name                                                               |                                                         Type                                                         | Description                                                    |
+|:-------------------------------------------------------------------|:--------------------------------------------------------------------------------------------------------------------:|:---------------------------------------------------------------|
+| `description` <span class="badge badge--secondary">Required</span> | [`SearchIndexDescription`](https://mongodb.github.io/node-mongodb-native/6.5/interfaces/SearchIndexDescription.html) | Description of the search index, including name and definition |
+
+`@searchIndex` is used to set search indices on the schema, this decorator acts like [`schema.searchIndex()`](https://mongoosejs.com/docs/api/schema.html#Schema.prototype.searchIndex()).
+
+:::note
+Because creating a search index can be a very heavy operation, automatic creation of search indices is disabled by
+default. To enable automatic creation of search indices, the [`autoSearchIndex`](https://mongoosejs.com/docs/guide.html#autoSearchIndex) option must be set to `true` in the
+schema options using the [`@modelOptions`](https://typegoose.github.io/typegoose/docs/api/decorators/model-options) decorator
+:::
```

Comment:
the atlas only note should be moved up here with either a `note`, `tip` or `caution` admonition, [test page](https://typegoose.github.io/typegoose/docs/test/#callouts--admonitions)